WindowsAppRuntime.sln

```diff
@@ -301,7 +301,11 @@ EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AppNotifications", "dev\AppNotifications\AppNotifications.vcxitems", "{B4824897-88E0-4927-8FB9-E60106F01ED9}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Windows.AppNotifications.Projection", "dev\Projections\CS\Microsoft.Windows.AppNotifications.Projection\Microsoft.Windows.AppNotifications.Projection.csproj", "{2385A443-A1C2-4562-8D28-D0AD239EE5E7}"
+EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common", "Common", "{82A73181-EA4A-431A-B82B-BE6734604CC9}"
+	ProjectSection(SolutionItems) = preProject
+		SDKPackageManager.ps1 = SDKPackageManager.ps1
```

Comment:
Relying on the .sln to perform test infrastructure work is strongly _not_ recommended.

We're currently using _VS_ as our project management process, but that has multiple problems today. We're identified we need to do something but haven't decided exactly what yet, and we've had more urgent work to focus on that we haven't address this yet. But as they say, it's not the end, but I can see it from here. Whether we stick with VS and define processes to make it manageable (e.g. projects we design for "I need a DLL" or "I need a test DLL" or... rather than the project templates VS has which are poor fits for our needs and time consuming + error prone to alter for our work). Or switch to msbuild (where *proj may be similar but *.sln doesn't exist). Or CMAKE. Or... We have thoughts but no specific decision other than _not what we're doing today_.

Also, the .sln is used in the build pipeline to compile our bits. We then send them over to Helix test VMs to run -- no .sln exists there.

Making our TAEF UT/FT self-contained and doing setup+cleanup in TAEF fixtures and using things like Test::Bootstrap::Setup() and like aids in `\test\inc\WindowsAppRuntime.Test.*` isa more reliable, performant and flexible solution.

I'm not quite following the intent here. What are you trying to accomplish? What is it about the test\inc etc support that you find inadequate?

Reply:
Notifications have their own apps where we test what a developer app may look like. In fact, Notifications app going into the samples repo is basically 1-1 with this inner loop dev app. And while unit testing gets us close, we can't be fully confident since we have multiple activation scenarios where we have to receive/activate a notification (raw/toast) from WNS, which we currently don't have automaton for. This E2E testing gives us confidence that if a user follows our API patterns, everything should be working properly when it makes it to the NuGet.